central/rbac/utils/cluster_permission_evaluator.go

```diff
@@ -52,16 +52,23 @@ func (c *clusterPermissionEvaluator) RolesForSubject(ctx context.Context, subjec
 
 func (c *clusterPermissionEvaluator) getBindingsAndRoles(ctx context.Context, subject *storage.Subject) ([]*storage.K8SRoleBinding, []*storage.K8SRole) {
 	q := search.NewQueryBuilder().
+		AddBoolsHighlighted(search.ClusterRole, true).
+		AddStringsHighlighted(search.RoleID, search.WildcardString).
 		AddExactMatches(search.ClusterID, c.clusterID).
-		AddBools(search.ClusterRole, true).
+		AddExactMatches(search.Namespace, "").
```

Comment:
Just want to make sure that you intend to specifically get those that do not have a namespace (as per our discussion to simplify the namespace check in `getRoleIDsFromBindings` by updating the queries).

Reply:
Yep exactly.